plugin.xml

```diff
@@ -15,14 +15,23 @@
 
   <js-module src="www/index.js" name="CordovaAndroidScreenshare">
     <clobbers target="CordovaAndroidScreenshare" />
+    <clobbers target="MediaProjectionService" />
   </js-module>
 
   <platform name="android">
     <config-file target="res/xml/config.xml" parent="/*">
       <feature name="CordovaAndroidScreenshare">
-        <param name="android-package" value="com.iotum.cordovaandroidscreenshare.CordovaAndroidScreenshare" />
+        <param name="android-package" value="com.iotum.CordovaAndroidScreenshare" />
       </feature>
     </config-file>
-    <source-file src="src/android/com/iotum/cordovaandroidscreenshare/CordovaAndroidScreenshare.java" target-dir="src/com/iotum/cordovaandroidscreenshare" />
+    <config-file parent="application" target="AndroidManifest.xml">
+      <service
+        android:name="com.iotum.MediaProjectionService"
+        android:enabled="true"
+        android:exported="false"
+        android:foregroundServiceType="mediaProjection" />
```

Comment:
This tells Android that its a MediaProjection foreground service, even if the service itself is empty and only has a notification.

Luckily we don't need to actually have the MediaProjection code in the service itself.